docs/src/versioning.md

```diff
@@ -0,0 +1,59 @@
+# Versioning policy
+
+This document describes the *Quality-time* versioning policy. It is aimed at *Quality-time* operators and developers.
+
+## Major, minor, and patch releases
+
+*Quality-time* complies with [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+A new **major** release of *Quality-time* is made when operators need to make changes to the Docker composition (besides upgrading version numbers) before upgrading *Quality-time*.
+
+A new **minor** release of *Quality-time* is made when the new version has new or changed functionality.
+
+A new **patch** release of *Quality-time* is made when the new version has only bug fixes.
+
+```{index} MongoDB
+```
+
+## Version overview
+
+The table below contains the *Quality-time* releases since the last minor of the previous major release. For each release it shows the release date, the MongoDB version included, the MongoDB feature compatibility (FC) version set in the database, whether migration code was added or removed, and whether up- or downgrading is possible, and if so, to which versions.
```

Comment:
Perhaps it is good to highlight that the statement "downgrading is possible" only refers to mongodb, or did you check all migrations?

Reply:
I checked all migrations! The table should show all up- and downgrade options for each v5 release.